src/features/pages/partners/deriv-prime/why-deriv-prime/types.ts

```diff
@@ -0,0 +1,6 @@
+import { TString } from 'types/generics'
+
+export type CardTypes = {
+    title?: TString
```

Comment:
title and subtitle gonne be there for sure right ? so can we remove **?** here 
```suggestion
    title: TString
```